crates/optimism/payload/src/builder.rs

```diff
@@ -625,13 +625,7 @@ where
         let base_fee = builder.evm_mut().block().basefee;
 
         while let Some(tx) = best_txs.next(()) {
-            // We skip invalid cross chain tx, they would be removed on the next block update in the
-            // maintenance job
-            if let Some(interop) = tx.interop() {
-                if !interop.is_valid(self.config.attributes.timestamp()) {
-                    continue
-                }
-            }
+            let interop = tx.interop();
```

Comment:
ah I overlooked the that we need to convert the tx, so we don't actually safe anything by moving the check down below

this is still fine

Reply:
Yes 😁
We save 1 instruction :)